### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by eparejatobes**
After #52 and specially #49 and #50, which where identified in #48, we are pretty happy with the current pipeline.

Let's release 0.11.0, I will check the data afterwards and then we can consider this stable.

### CODE REVIEW

**Comment by laughedelic** [comment]
OK, I'm assigned, so I guess, I have to generate the data.

**Comment by eparejatobes** [comment]
I don't want to delay this, I'm doing it myself.

**Comment by laughedelic** [comment]
I was going to merge #47 and run it now. So if you want to do it, please, merge #47 first.

**Comment by eparejatobes** [comment]
I'm not merging #47, it's untested and dealing with iterators is tricky. What we have now works, and I'm much more interested in getting the database right than in the performance/hardware requirements.

File: src/test/scala/pick16SCandidates.scala [hunk 1]
```diff
@@ -116,7 +116,7 @@ case object pick16SCandidates extends FilterData(
         sys.error(s"ID [${commonID}] is not found in the FASTA. Check RNACentral filtering.")
 
       val (acceptedRows, rejectedRows) =
-        if ( sequencePredicate(fasta.getV(sequence)) ) {
+        if ( sequencePredicate( (fasta get sequence).value ) ) {
```
Comment (eparejatobes) [inline src/test/scala/pick16SCandidates.scala:119]: that's why I want to disable that name hashing thing.

**Comment by eparejatobes** [comment]
couldn't publish the release on github, see ohnosequences/nice-sbt-settings#53

**Comment by laughedelic** [comment]
Replied there.

**Comment by laughedelic** [comment]
Sorry, it's my fault: [here](https://github.com/ohnosequences/db.rna16s/blob/master/build.sbt#L12), it should be

``` scala
GithubRelease.repo := s"${organization.value}/${name.value}"
```

UPD: fixed.

**Comment by eparejatobes** [comment]
Is there a way of publishing the release on github now?

**Comment by laughedelic** [comment]
Yes

``` scala
> reload
> set version := "0.11.0"
> releaseOnGithub
```

**Comment by laughedelic** [comment]
@eparejatobes regarding those two suspicious _Leptospira fainei_ guys, here are the values from the blast output (cut IDs prefix, transposed):

| column | parent | child |
| :-: | :-- | :-- |
| qseqid | URS0000424C5E | URS0000043A26 |
| qstart | 1 | 1 |
| qend | 1466 | 1453 |
| qlen | 1481 | 1464 |
| sseqid | URS0000299BD4 | URS0000299BD4 |
| sstart | 1 | 14 |
| send | 1466 | 1466 |
| slen | 1466 | 1466 |
| evalue | 0.0 | 0.0 |
| score | 1450 | 1453 |
| bitscore | 2791 | 2794 |
| length | 1466 | 1453 |
| pident | 99.59 | 100.00 |
| mismatch | 6 | 0 |
| gaps | 0 | 0 |
| gapopen | 0 | 0 |
| qcovs | 99 | 99 |

I think parent gets discarded because its `pident` is filtered out as non-maximum.

**Comment by eparejatobes** [comment]
Data is generated, release published. Merging, doing data review elsewhere.

**Comment by laughedelic** [comment]
@eparejatobes are you going to make another release or we will use this one for testing MG7?

**Pull request merged**